packages/noports_core/lib/src/sshrvd/build_env.dart

```diff
@@ -1,4 +1,5 @@
+import 'dart:io';
+
 class BuildEnv {
-  static final bool enableSnoop =
-      bool.fromEnvironment('ENABLE_SNOOP', defaultValue: false);
+  static final bool enableSnoop = (Platform.environment['ENABLE_SNOOP'] ?? "false").toLowerCase() == 'true';
```

Comment:
I found that `bool.fromEnvironment` was not working properly on MacOS

Reply:
We should probably file a bug with the dart team